.github/PULL_REQUEST_TEMPLATE/version_bump_pr_template.md

```diff
@@ -0,0 +1,13 @@
+### Acceptance Criteria
+- Include here all things that this PR should solve
```

Comment:
Should we change this to something like: "Include links to all PRs that will be included in this version"?

Reply:
I think we should, done in 6c728840a6320d744fcba7207ad68dae69edd281